airflow/migrations/versions/092435bf5d12_add_max_active_runs_column_to_dagmodel_.py

```diff
@@ -0,0 +1,58 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Add max_active_runs column to dagmodel table
+
+Revision ID: 092435bf5d12
+Revises: 142555e44c17
+Create Date: 2021-09-06 21:29:24.728923
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy import text
+
+# revision identifiers, used by Alembic.
+revision = '092435bf5d12'
+down_revision = '7b2661a43ba3'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    """Apply Add max_active_runs column to dagmodel table"""
+    op.add_column('dag', sa.Column('max_active_runs', sa.Integer(), nullable=True))
+    with op.batch_alter_table('dag_run', schema=None) as batch_op:
+        # Add index to dag_run.dag_id and also add index to dag_run.state where state==running
+        batch_op.create_index('idx_dag_run_dag_id', ['dag_id'])
+        batch_op.create_index(
+            'idx_dag_run_running_dags',
+            ["state", "dag_id"],
+            postgres_where=text("state='running'"),
+            mssql_where=text("state='running'"),
```

Comment:
let's add `sqlite_where` too

Reply:
https://docs.sqlalchemy.org/en/14/dialects/sqlite.html#partial-indexes

Reply:
Wow mysql really isn't a good database is it 😉